### PR TITLE
Ftv 145 shared normals

### PR DESCRIPTION
**Pull request by clusty**
This is the same PR as Chris' but with a cleaner history and some dead code cleanup.

* Calculate normals on original topology
* Remap normals to refined mesh
* Linux name fix

### CODE REVIEW

File: Source/abci/Importer/aiObject.cpp [hunk 1]
```diff
@@ -58,8 +58,13 @@ aiObject::aiObject(aiContext *ctx, aiObject *parent, const abcObject &abc)
     , m_abc(abc)
     , m_parent(parent)
 {
+#if defined(__linux__)
```
Comment (mfe) [inline Source/abci/Importer/aiObject.cpp]: Would have been nice to have this in a separate commit

File: Source/abci/pch.h [hunk 2]
```diff
@@ -57,6 +57,7 @@ using namespace Alembic;
 
 using abcV2 = Imath::V2f;
 using abcV3 = Imath::V3f;
+using abcV3i = Imath::V3i;
```
Comment (mfe) [inline Source/abci/pch.h]: Not sure where this is used ?

Comment (clusty) [reply Source/abci/pch.h]: Doubt it's needed.

File: Source/abci/Importer/aiObject.cpp (hunk 1 above)
Comment (clusty) [reply Source/abci/Importer/aiObject.cpp]: Done

**Pull request merged**